nswag/Client.Class.ReadObjectResponse.liquid

```diff
@@ -0,0 +1,60 @@
+{
+    if (response == null || response.Content == null)
+    {
+{%- if GenerateNullableReferenceTypes -%}
+        return new ObjectResponseResult<T>(default(T)!, string.Empty);
+{%- else -%}
+        return new ObjectResponseResult<T>(default(T), string.Empty);
+{%- endif -%}
+    }
+
+    if (ReadResponseAsString)
+    {
+        var responseText = await response.Content.ReadAsStringAsync().ConfigureAwait(false);
+        try
+        {
+            var typedBody = {% if UseSystemTextJson %}System.Text.Json.JsonSerializer.Deserialize{% else %}Newtonsoft.Json.JsonConvert.DeserializeObject{% endif %}<T>(responseText, {% if UseRequestAndResponseSerializationSettings %}Response{% endif %}JsonSerializerSettings);
+{%- if GenerateNullableReferenceTypes -%}
+            return new ObjectResponseResult<T>(typedBody!, responseText);
+{%- else -%}
+            return new ObjectResponseResult<T>(typedBody, responseText);
+{%- endif -%}
+        }
+        catch ({% if UseSystemTextJson %}System.Text.Json.JsonException{% else %}Newtonsoft.Json.JsonException{% endif %} exception)
+        {
+            var message = "Could not deserialize the response body string as " + typeof(T).FullName + ".";
+            throw {{ ExceptionClass }}.Create((int)response.StatusCode, headers, responseText, JsonSerializerSettings, exception);
+        }
+    }
+    else
+    {
+        try
+        {
+            using (var responseStream = await response.Content.ReadAsStreamAsync().ConfigureAwait(false))
+{%- if UseSystemTextJson -%}
+            {
+                var typedBody = await System.Text.Json.JsonSerializer.DeserializeAsync<T>(responseStream, {% if UseRequestAndResponseSerializationSettings %}Response{% endif %}JsonSerializerSettings, cancellationToken).ConfigureAwait(false);
+{%- else -%}
+            using (var streamReader = new System.IO.StreamReader(responseStream))
+            using (var jsonTextReader = new Newtonsoft.Json.JsonTextReader(streamReader))
+            {
+                var serializer = Newtonsoft.Json.JsonSerializer.Create({% if UseRequestAndResponseSerializationSettings %}Response{% endif %}JsonSerializerSettings);
+                var typedBody = serializer.Deserialize<T>(jsonTextReader);
+{%- endif -%}
+{%- if GenerateNullableReferenceTypes -%}
+                return new ObjectResponseResult<T>(typedBody!, string.Empty);
+{%- else -%}
+                return new ObjectResponseResult<T>(typedBody, string.Empty);
+{%- endif -%}
+            }
+        }
+        catch ({% if UseSystemTextJson %}System.Text.Json.JsonException{% else %}Newtonsoft.Json.JsonException{% endif %} exception)
+        {
+            var message = "Could not deserialize the response body stream as " + typeof(T).FullName + ".";
+            throw {{ ExceptionClass }}.Create((int)response.StatusCode, headers, exception);
```

Comment:
This line handles the case when throwing from unable to deserialize the response and a JsonException is thrown. An `ApiException` with a `ProblemDetails` with minimal information will be thrown. See this commit to view what changes this line made to the generated code.
https://github.com/Laserfiche/lf-repository-api-client-dotnet/pull/61/commits/67b69741f1c0e5cf539a89cca07c22432650c4af

The original template file can be found here
https://github.com/RicoSuter/NSwag/blob/master/src/NSwag.CodeGeneration.CSharp/Templates/Client.Class.ReadObjectResponse.liquid